Project1/ExtendedCalculator.cs

```diff
@@ -56,6 +56,52 @@ public int GetPowerOfNumber(int number1, int number2)
             }
             return result;
         }
+        public int GetAverageOfNumbers(int[] number, int amoungOfNumber)
+        {
+            for (int i = 0; i < amoungOfNumber; i++)
```

Comment:
Nie powinniśmy tego fora dawać w metodzie, ona powinna tylko przyjąć gotową tablicę i zrobić na niej działania a nie odpowiadać ze uzupełnienie tablicy wartościami. To powinno byc w program.cs